fix/fix_am/global_hyblev_fcst_rrfsL65.txt

```diff
@@ -0,0 +1,68 @@
+     2    66
+       0.000  1.00000000
+       0.000  0.99787363
+       0.000  0.99547387
+       0.017  0.99276698
+       0.234  0.98971388
+       1.058  0.98626929
+       3.103  0.98238159
+       7.211  0.97799245
+      14.450  0.97303658
+      26.130  0.96744145
+      43.805  0.96112727
+      69.754  0.95400221
+     107.500  0.94595747
+     160.901  0.93687684
+     233.980  0.92663902
+     330.883  0.91511905
+     455.809  0.90219033
+     612.932  0.88772712
+     806.289  0.87160769
+    1039.659  0.85371818
+    1316.413  0.83395695
+    1639.350  0.81223965
+    2010.516  0.78850460
+    2431.016  0.76271850
+    2900.827  0.73488203
+    3418.626  0.70503512
+    3981.638  0.67326135
+    4585.535  0.63969119
+    5224.384  0.60450347
+    5890.665  0.56792486
+    6575.364  0.53022703
+    7268.150  0.49172137
+    7957.623  0.45275150
+    8631.631  0.41368369
+    9277.633  0.37489594
+    9883.087  0.33676625
+   10435.842  0.29966091
+   10924.498  0.26392352
+   11338.722  0.22986547
+   11669.502  0.19775844
+   11909.318  0.16782909
+   12052.236  0.14025627
+   12093.928  0.11517042
+   12031.620  0.09265514
+   11863.989  0.07275035
+   11591.011  0.05545674
+   11213.791  0.04074101
+   10734.376  0.02854147
+   10155.564  0.01877378
+    9480.727  0.01133629
+    8714.452  0.00610706
+    7877.419  0.00279793
+    7004.872  0.00097247
+    6132.608  0.00018806
+    5296.374  0.00000379
+    4530.025  0.00000000
+    3843.468  0.00000000
+    3229.725  0.00000000
+    2681.893  0.00000000
+    2193.544  0.00000000
+    1758.738  0.00000000
+    1372.014  0.00000000
+    1028.378  0.00000000
+     723.285  0.00000000
+     452.613  0.00000000
+     212.637  0.00000000
+       0.000  0.00000000
```

Comment:
This file is added because that is what `RRFS_dev1` uses but currently not activated. It will be activated in later PRs as it changes levels from `64` to `65`